sql/migrations/20200706205532_world.sql

```diff
@@ -0,0 +1,121 @@
+(94750, 1064, 0, 0, 0, 0, 0, 0, -12374, 184.474, 3.24808, 3.34054, 300, 300, 0, 100, 0, 0, 0, 0, 0, 10);
+
+-- add correct waypoints to grom gol grunt
+DELETE FROM `creature_movement` WHERE `id` = 610;
+INSERT INTO `creature_movement` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `waittime`, `wander_distance`, `script_id`) VALUES
+(610, 1, -12383.7, 227.084, 2.67865, 100, 13000, 0, 0),
+(610, 2, -12384.2, 224.195, 2.64434, 100, 0, 0, 0),
+(610, 3, -12383.3, 215.205, 2.3636, 100, 0, 0, 0),
+(610, 4, -12383.4, 206.133, 2.02113, 100, 0, 0, 0),
+(610, 5, -12383.7, 203.173, 1.99872, 100, 0, 0, 0),
+(610, 6, -12385, 200.433, 2.13383, 100, 0, 0, 0),
+(610, 7, -12386.6, 197.907, 2.20216, 100, 0, 0, 0),
+(610, 8, -12391.9, 195.553, 1.84427, 3.03687, 78000, 0, 0),
+(610, 9, -12386.6, 197.907, 2.20216, 100, 0, 0, 0),
+(610, 10, -12385, 200.433, 2.13383, 100, 0, 0, 0),
+(610, 11, -12383.7, 203.173, 1.99872, 100, 0, 0, 0),
+(610, 12, -12383.4, 206.133, 2.02113, 100, 0, 0, 0),
+(610, 13, -12383.3, 215.205, 2.3636, 100, 0, 0, 0),
+(610, 14, -12384.2, 224.195, 2.64434, 100, 0, 0, 0);
+
+-- commander aggro gosh should roam
+UPDATE `creature` SET `movement_type` = 1, `wander_distance` = 3 WHERE `guid` = 721;
+
+
+-- rebel camp
+-- add rebel watchman waypoints
+UPDATE `creature` SET `movement_type` = 2 WHERE `id` = 754;
+INSERT INTO `creature_movement_template` (`entry`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `waittime`, `wander_distance`, `script_id`) VALUES
+(754, 1, -11324, -213.783, 76.5032, 2.25148, 87000, 0, 0),
+(754, 2, -11327.2, -212.021, 75.9563, 100, 0, 0, 0),
+(754, 3, -11334.8, -209.889, 75.2551, 100, 0, 0, 0),
+(754, 4, -11342.8, -210.905, 75.2215, 100, 0, 0, 0),
+(754, 5, -11346.8, -210.962, 75.2223, 100, 0, 0, 0),
+(754, 6, -11354.8, -210.423, 75.2585, 100, 0, 0, 0),
+(754, 7, -11366.8, -212.733, 75.2209, 100, 0, 0, 0),
+(754, 8, -11370.6, -213.99, 75.2973, 100, 0, 0, 0),
+(754, 9, -11376.5, -219.242, 75.0468, 100, 0, 0, 0),
+(754, 10, -11379.7, -226.428, 73.4426, 100, 0, 0, 0),
+(754, 11, -11381.6, -237.335, 68.7688, 100, 0, 0, 0),
+(754, 12, -11382.6, -248.607, 65.2183, 100, 25000, 0, 0),
+(754, 13, -11381.9, -240.936, 67.6475, 100, 0, 0, 0),
+(754, 14, -11379.7, -229.946, 72.1583, 100, 0, 0, 0),
+(754, 15, -11378.7, -226.508, 73.5829, 100, 0, 0, 0),
+(754, 16, -11375.4, -219.386, 75.0807, 100, 0, 0, 0),
+(754, 17, -11373, -216.171, 75.2465, 100, 0, 0, 0),
+(754, 18, -11365.7, -212.828, 75.2215, 100, 0, 0, 0),
+(754, 19, -11353.7, -211.491, 75.2339, 100, 0, 0, 0),
+(754, 20, -11345.6, -211.416, 75.2236, 100, 0, 0, 0),
+(754, 21, -11337.5, -210.781, 75.2219, 100, 0, 0, 0),
+(754, 22, -11333.4, -210.355, 75.283, 100, 0, 0, 0);
+
+-- add private merle waypoints

```

Comment:
Are you sure that this NPC have OOC movement?
https://github.com/vmangos/core/commit/e283993180f7fe4b7601bf265f8ab7d6f8159a5f

Reply:
He just patrols the camp constantly, just checked classic.

Reply:
He does have a script where he runs out of the camp with the rebel watchman chasing him. Not sure what that is.